changelog.d/746.bugfix.rst

```diff
@@ -0,0 +1 @@
+Add build requirements to pyproject.toml. (gh pr #746)
```

Comment:
Unless you don't care about getting credit in the changelog, I would word this as:

```
Add build requirements to pyproject.toml. Fixed by @anthrotype (gh issue #736, gh pr #746)
```

If you'd like to retain your commit signatures, please edit the commit history (or make a fixup commit and squash it when you are satisfied with the result).

Reply:
Either way you should add the issue number.